samples/MQTTSN/src/paho_iot_hub_telemetry_example_qos_0.c

```diff
@@ -0,0 +1,265 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#define WARN printf
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>

```

Comment:
nit: lets sort these std lib header files

Generally I follow following:

dir2/foo2.h.
A blank line
C system headers (more precisely: headers in angle brackets with the .h extension), e.g., <unistd.h>, <stdlib.h>.
A blank line
C++ standard library headers (without file extension), e.g., <algorithm>, <cstddef>.
A blank line
Other libraries' .h files.
Your project's .h files.

Reply:
what is `dir2/foo2.h.` referring to, here?